content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md

```diff
@@ -0,0 +1,202 @@
+# Automated High Availability in kubeadm v1.15: Batteries Included But Swappable
```

Comment:
This blog post needs usual front matter. Have a look at other blog posts for an example.